libexec/scoop-shim.ps1

```diff
@@ -12,7 +12,7 @@
 #
 # To list all shims or matching shims, use the 'list' subcommand:
 #
-#     scoop shim list [<shim_name>/<pattern>...]
+#     scoop shim list [<shim_name>/<regex_pattern>...]
```

Comment:
```suggestion
#     scoop shim list [<regex_pattern>...]
```

To clarify, only use `regex pattern` here. I've just found `scoo*` matches 'scoop', 'score', etc. and `7z*` matches `^7` here, lol.

Reply:
OK.